.gitignore

```diff
@@ -9,3 +9,4 @@ data/wiki_specialphrases.sql
 data/osmosischange.osc
 
 .vagrant
+.vscode
```

Comment:
We don't take editor-specific ignores here. Those should usually go into the .gitignorefile in your home directory.